app/experimenter/experiments/constants.py

```diff
@@ -421,6 +422,8 @@ class EmailType(models.TextChoices):
     ERROR_FIREFOX_VERSION_MAX = (
         "Ensure this value is greater than or equal to the minimum version"
     )
+    ERROR_FEATURE_ENABLED = f"Feature enabled is only supported for\
+         {FEATURE_ENABLED_MIN_UNSUPPORTED_VERSION[Application.DESKTOP]} version and less"
```

Comment:
Oh yeah, I see what you mean.  To match the language we're using for the mobile, we might say it something more like

```
Experiments for Firefox Desktop {version} and above must have the feature enabled and specify a feature value.
```